installer/install_scripts/sshprep

```diff
@@ -246,8 +246,13 @@ check_config_block() {
 					status "Have an ipv6 address only for $2"
 					prepend_config_block "\nHost ${2} ${2}-v6 ${ip6}\n\tHostname\t\t${ip6}\n\tHostKeyAlias\t\t${2}\n${user_line}\n"
 				else
-					status "No ipv4 or ipv6 address found for ${2}.  Please enter an IP address to use:"
-					read ip_or_hostname
+					#echo "== No ipv4 or ipv6 address found for ${2}.  Please enter an IP address to use:" >&2
+					#read ip_or_hostname <&2
+					#if [ -z "$ip_or_hostname" ]; then
+					#	ip_or_hostname="$2"
+					#fi
```

Comment:
Is there a reason that this is commented out? If it's safe to remove, we may want to do so unless you think it'll be something that we find useful to add back in in the future.

Reply:
You have that exactly right.  The short term fix to get rita out was to disable this code.  Once ACH is out as well it'll be easier to revisit this.